src/routes/safe/components/Layout/Tabs/index.tsx

```diff
@@ -6,53 +6,82 @@ import { withRouter } from 'react-router-dom'
 
 import { styles } from './style'
 
-import {
-  ADDRESS_BOOK_TAB_BTN_TEST_ID,
-  BALANCES_TAB_BTN_TEST_ID,
-  SETTINGS_TAB_BTN_TEST_ID,
-  APPS_TAB_BTN_TEST_ID,
-  TRANSACTIONS_TAB_BTN_TEST_ID,
-} from 'src/routes/safe/components/Layout'
 import SettingsTab from 'src/routes/safe/components/Layout/Tabs/SettingsTab'
 import { AddressBookIcon } from 'src/routes/safe/components/assets/AddressBookIcon'
 import { AppsIcon } from 'src/routes/safe/components/assets/AppsIcon'
 import { BalancesIcon } from 'src/routes/safe/components/assets/BalancesIcon'
 import { TransactionsIcon } from 'src/routes/safe/components/assets/TransactionsIcon'
 
+export const BALANCES_TAB_BTN_TEST_ID = 'balances-tab-btn'
+export const SETTINGS_TAB_BTN_TEST_ID = 'settings-tab-btn'
+export const APPS_TAB_BTN_TEST_ID = 'apps-tab-btn'
+export const TRANSACTIONS_TAB_BTN_TEST_ID = 'transactions-tab-btn'
+export const TRANSACTIONS_TAB_NEW_BTN_TEST_ID = 'transactions-tab-new-btn'
+export const ADDRESS_BOOK_TAB_BTN_TEST_ID = 'address-book-tab-btn'
+
 interface Props {
   classes: Record<string, any>
   match: Record<string, any>
   history: Record<string, any>
   location: Record<string, any>
```

Comment:
would it be possible to also remove `any` from there?

Reply:
should be better now, I couldn't find good approach to type `classes`, if you have any suggestion just tell me